packages/filebrowser-extension/src/index.ts

```diff
@@ -428,9 +428,14 @@ function addCommands(
   commands.addCommand(CommandIDs.navigate, {
     execute: args => {
       const path = (args.path as string) || '';
+      const type = (args.type as string) || '';
       return Private.navigateToPath(path, factory)
         .then(() => {
-          return commands.execute('docmanager:open', { path });
+          // If executed explicitly on a directory, do nop after navigateToPath
+          // If no type provided, execute docmanager:open as before
```

Comment:
Add an `else` clause below, ie `else {return;}`. That way the code itself will be easy to read, and then you can get rid of the comments above.